lib/index.js

```diff
@@ -773,7 +773,7 @@ Dataviz.prototype.render = function(results = undefined){
               visibilityChange = "webkitvisibilitychange";
             }
             const handleVisibilityChange = () => {
-              if(!document[hidden]) {
+              if(!document[hidden] && typeof datavizInstance.view._artifacts.c3 !== 'undefined') {
```

Comment:
please update this statement to `if(!document[hidden] && datavizInstance.view._artifacts.c3)`

Reply:
Done! 😊